src/modules/translation/languages/zh.json

```diff
@@ -943,6 +945,11 @@
   },
   "third_party_collection_detail_page": {
     "notice": "每个发布的项目需要 1 个插槽。 {buy_link} 购买老虎机包并开始发布您的作品！",
+    "search_placeholder": "搜索 {count} {count, plural, one {item} other {items}}...",
```

Comment:
We seem not to be using the word "item" for item in Chinese. The same goes for the selection property.

Reply:
True!